charms/scripts/update-machine-charm.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 DASHBOARD_VERSION=${1:-latest}
-dist_path=../../charms/machine-charm/src/dist
+dist_path=../machine-charm/src/dist
```

Comment:
This works when called in action.yml, but not when called from the root dir like it is in the docs. If you get the absolute location of this script and then add `../machine-charm/src/dist` to it then it will work regardless of the location this script gets called from.